src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -51,6 +51,7 @@
 
     private final String replacementStrategy;
     private final boolean taggable;
+    private final Map<String, Object> tagging = new HashMap<>();
```

Comment:
We should model the `tagging` as a new class, e.g. `ResourceTagging`  instead of `Map<String, Object>`, it's hard to know what inside the Map.

Reply:
sounds good, yeah i can revise it